### DISCUSSION TITLE
Tier upload and download limit ???

### DISCUSSION THREAD

**Discussion by Hower-Pan** [seaweedfs/seaweedfs#2192] [Q&A]
I config master.toml backend to other s3 storage ,I tier upload  some collection to s3 
collection file object count over 500K tier upload and download data will be lose  
tire upload below message
<img width="468" alt="image" src="https://user-images.githubusercontent.com/33602331/125311589-1ac1a600-e366-11eb-979f-7f06df61b93d.png">
final it's finished upload use http to access  collection object file , page return not found
<img width="468" alt="image" src="https://user-images.githubusercontent.com/33602331/125312233-adfadb80-e366-11eb-91b3-8a2b7cc3bb7d.png">

try to tier download recovery it, but download fail
<img width="468" alt="image" src="https://user-images.githubusercontent.com/33602331/125312691-15189000-e367-11eb-86a2-df0597e1374e.png">

I don't know what's  wrong with it.  have any suggest for me to debug it...    

Thank you.

**Comment by chrislusf** [comment]
can you check what has been uploaded to the s3 bucket?

**Reply by Hower-Pan** [reply]
that is my result , but  I try use 2.59 it's workable thank you

2.57
![image](https://user-images.githubusercontent.com/33602331/126089355-f0c29ec4-3ee2-4999-9547-505a9dd0128f.png)

2.59
<img width="771" alt="image" src="https://user-images.githubusercontent.com/33602331/126089500-df70f8ee-7913-4acf-92ab-30331d5e10c9.png">

**Reply by chrislusf** [reply]
thanks for the update! Do you upload to the S3 or some other storage?

**Reply by Hower-Pan** [reply]
No I do not upload to AWS S3
because I try upload to my ISP S3 service fail , credentials is correct

<img width="1054" alt="image" src="https://user-images.githubusercontent.com/33602331/126869208-02410adc-6ed7-4c59-9ba0-e81587aa53a8.png">

I don't how debug it so need find solution to solve issue (ex:s3fs or goofys)
